### PR TITLE
checking whether the value is finite when findSegmentIndex_ for Tabulated1DFunction.

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
@blattms  do you have time to have a look at this and let the person reported the issue to report a little more details regarding the issue?

File: opm/material/common/Tabulated1DFunction.hpp [hunk 1]
```diff
@@ -455,8 +455,14 @@ class Tabulated1DFunction
...
     {
+        if (!isfinite(x)) {
+            std::ostringstream sstream;
+            sstream << "We can not search for extrapolation/interpolation segment in an 1D table for non-finite value " << getValue(x) << " .";
+            throw std::runtime_error(sstream.str());
+        }
+
         if (!extrapolate && !applies(x))
-            throw std::logic_error("Tried to evaluate a tabulated function outside of its range");
+            throw std::logic_error("Tring to evaluate a tabulated function outside of its range");
```
Comment (atgeirr) [inline opm/material/common/Tabulated1DFunction.hpp:465]: Typo

**Review by atgeirr — APPROVED** [review]
Good to mrege (when typo fixed).

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
Approved and green. I am merging it in.

**Pull request merged**